pyproject.toml

```diff
@@ -119,6 +119,8 @@ ignore = [
     # No explicit `stacklevel` keyword argument found
     "B028",
 
+    # Return condition directly, sometimes the error message isn't correct.
```

Comment:
I don't think the error message is incorrect, it's just not as clear as it could be - it doesn't explicitly say that you might need to negate the condition in some cases. But we can leave this comment as-is if you prefer :)

I think the actual reason we are skipping this is that we believe that sometimes it's more readable the other way.